### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Gl0deanR**
I have done some text corrections.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,13 +6,13 @@ A chrome extension to download ebooks directly from Goodreads. Fetches ebooks fr
 ## How to install
 1. Clone this repository.
 2. Open chrome extensions panel by visiting `chrome://extensions`.
-3. Turn developer mode on.
+3. Turn on developer mode.
```
Comment (laxyapahuja) [inline README.md:9]: ?

File: README.md [hunk 2]
```diff
@@ -6,13 +6,13 @@ A chrome extension to download ebooks directly from Goodreads. Fetches ebooks fr
...
 4. Click on Load Unpacked.
 5. Select the folder containing the repository with the root directory.
-6. Visit any books' page on Goodreads. The ebooks should load under Related Books section.
+6. Visit any books' page on Goodreads. The ebooks should load under the Related Books section.
```
Comment (laxyapahuja) [inline README.md:12]: unnecessary

File: README.md [hunk 3]
```diff
@@ -6,13 +6,13 @@ A chrome extension to download ebooks directly from Goodreads. Fetches ebooks fr
...
 7. Choose different sources according to your needs.
 
 ## How to contribute
-Help contribute by using it, finding bugs and opening issues. Pull requests are also appreciated!
+Help contribute by using it, finding bugs, and opening issues. Pull requests are also appreciated!
```
Comment (laxyapahuja) [inline README.md:16]: unnecessary

File: README.md [hunk 4]
```diff
@@ -6,13 +6,13 @@ A chrome extension to download ebooks directly from Goodreads. Fetches ebooks fr
...
 
-API repository available at [laxyapahuja/bunken-api](https://github.com/laxyapahuja/bunken-api).
\ No newline at end of file
+API repository is available at [laxyapahuja/bunken-api](https://github.com/laxyapahuja/bunken-api).
```
Comment (laxyapahuja) [inline README.md:18]: unnecessary

**Comment by Gl0deanR** [comment]
Just some simple/minor changes, I have some friends that use this so I thought I could give a hand in the readme. I'll close this down now. Sorry if I made a mistake.

**Pull request closed without merging**